api/DHSC.FingertipsNext.Modules.HealthData.UnitTests/Controllers/V1/IndicatorsControllerTests.cs

```diff
@@ -144,7 +146,7 @@ public async Task GetIndicatorData_ReturnsBadResponse_WhenMoreThan10CodesSupplie
     {
         var response = await _controller.GetIndicatorDataAsync(3,
             ["areaCode1", "ac2", "ac3", "ac4", "ac5", "ac6", "ac7", "ac8", "ac9", "ac10", "ac11"], "",
-            [1]) as BadRequestObjectResult;
+            years:[1]) as BadRequestObjectResult;
 
         response?.StatusCode.ShouldBe(400);
         (response?.Value as SimpleError)?.Message.Contains("Too many values supplied for parameter area_codes").ShouldBeTrue();
```

Comment:
Do we need an additional test for the badRequest for when area_group isn't provided as expected?